backend/package.json

```diff
@@ -14,26 +15,49 @@
     "crypto": "0.0.3",
     "express": "4.15.2",
     "express-session": "1.15.3",
+    "mkdirp": "^0.5.1",
     "nodemon": "1.11.0",
     "passport": "0.3.2",
     "passport-local": "1.0.0",
     "reflect-metadata": "0.1.10",
     "sequelize": "3.30.4",
     "sequelize-typescript": "0.3.1",
+    "sqlite": "^2.8.0",
     "source-map-support": "0.4.15",
     "ts-loader": "2.0.3",
     "typescript": "2.3.2",
     "webpack": "2.5.1"
   },
   "devDependencies": {
-    "concurrently": "3.4.0",
+    "@types/jest": "19.2.4",
+    "jest": "20.0.4",
     "nodemon": "1.11.0",
-    "prettier": "1.4.4",
     "sqlite3": "3.1.8",
+    "supertest": "3.0.0",
+    "ts-jest": "20.0.6",
+    "prettier": "1.4.4",
     "tslint": "5.2.0",
     "tslint-loader": "3.5.3"
   },
+  "engines": {
+    "node": "8.0.0"
```

Comment:
Just to make sure... Our code will not work with Node 7? Because then we should also advise Heroku to use Node 7.